run.py

```diff
@@ -1273,6 +1271,19 @@ def custom_openapi():
         "--setting_file", type=Path, default=USER_SETTING_PATH, help="設定ファイルを指定できます。"
     )
 
+    parser.add_argument(
+        "--preset_file",
+        type=Path,
+        default=None,
+        help=(
+            "プリセットファイルを指定できます。"
+            "指定がない場合、環境変数 VV_PRESET_FILE から読み込みます。"
+            "VV_PRESET_FILEの指定がない場合、--voicevox_dirのpresets.yamlから読み込みます。"
+            "voicevox_dirの指定がない場合、実行ファイルのディレクトリのpresets.yamlから読み込みます。"
+            "指定されたパスにファイルが存在しないまたは破損している場合、プリセットAPIはエラー（HTTP 422）を返します。"
```

Comment:
細かいドキュメント良いですね！
ただ、ちょっと粒度がほかの引数と揃ってない気がしました。長すぎるhelpは他のhelpの可読性を下げてhelp本来の役割をはたしづらいかも･･･？

最後の一文はAPI側のドキュメントに移して、多少不正確でも３行め４行めをこんな感じでまとめちゃうのどうでしょう。

```suggestion
            "プリセットファイルを指定できます。"
            "指定がない場合、環境変数 VV_PRESET_FILE、--voicevox_dirのpresets.yaml、実行ファイルのディレクトリのpresets.yamlを順に探します。"
```


（起動コマンド引数に関して、`docs/`あたりに詳細を書いても良いかもとちょっと思いました。）

Reply:
> 粒度が揃っていない

たしかにです！　suggestionを適用します。

> 最後の一文はAPI側のドキュメントに移して

プリセットAPIにHTTPステータスごとの説明文（`responses={ 422: { "description": "hogehoge" }}`）を追加すればよさそうですね。

プリセットAPIのHTTP 422は、プリセットファイルの不存在・破損以外の要因で失敗した場合にも返ります。
このPRで発生しやすくなりますが、ちょっと範囲が広いのでいったんIssue化しました。

- https://github.com/VOICEVOX/voicevox_engine/issues/760

別PRで対応して、このPRでは最後の一文は含めない方向で考えています。

> 起動コマンド引数に関して、docs/あたりに詳細を書いても良いかも

他の引数でも対応したいので、Issue化しました！

- https://github.com/VOICEVOX/voicevox_engine/issues/759